### PR TITLE
Sovereign accounts and EVM system layer

### PR DESCRIPTION
**Pull request by dmitrylavrenov**

### CODE REVIEW

File: frame/evm-system/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+description = "FRAME EVM SYSTEM pallet."
+authors = { workspace = true }
+edition = { workspace = true }
+repository = { workspace = true }
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+log = { version = "0.4.17", default-features = false }
```
Comment (MOZGIII) [inline frame/evm-system/Cargo.toml:14]: Should probably use workspace dep here as well

File: frame/evm-system/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+scale-codec = { package = "parity-scale-codec", workspace = true }
+scale-info = { workspace = true }
+# Substrate
+frame-support = { workspace = true }
+frame-system = { workspace = true }
+sp-runtime = { workspace = true }
+sp-std = { workspace = true }
+
+[dev-dependencies]
+mockall = "0.11"
```
Comment (MOZGIII) [inline frame/evm-system/Cargo.toml:24]: https://github.com/humanode-network/frontier/pull/76/files#r1364841504 also here

**Review by MOZGIII — APPROVED** [review]
Looks good!

File: frame/evm-system/Cargo.toml (hunk 2 above)
Comment (dmitrylavrenov) [reply frame/evm-system/Cargo.toml]: https://github.com/humanode-network/frontier/pull/76/commits/96dbde2bbb3adfac8c82d7b5f298b9085bcaf81c

File: frame/evm-system/Cargo.toml (hunk 1 above)
Comment (dmitrylavrenov) [reply frame/evm-system/Cargo.toml]: https://github.com/humanode-network/frontier/pull/76/commits/96dbde2bbb3adfac8c82d7b5f298b9085bcaf81c

**Pull request merged**